src/MatrixAlgebraKit.jl

```diff
@@ -34,6 +34,7 @@ export LAPACK_HouseholderQR, LAPACK_HouseholderLQ, LAPACK_Simple, LAPACK_Expert,
     LAPACK_QRIteration, LAPACK_Bisection, LAPACK_MultipleRelativelyRobustRepresentations,
     LAPACK_DivideAndConquer, LAPACK_Jacobi
 export LQViaTransposedQR
+export PolarViaSVD, PolarNewton
```

Comment:
Definitely agree that we should export this, but before we explicitly mark this as public, I noticed that we have:
`PolarViaSVD(svdalg)` and `LQViaTransposedQR(qr_alg)` (note the underscore in the second).
I'm fine with either, but we should probably be consistent, and this might be the correct time to do that?

Reply:
Either way is fine with me.